docs/src/pages/components/Images/WebImages.mdx

```diff
@@ -4,19 +4,19 @@ platform: web
 githubPath: bpk-component-image
 ---
 
-import Readme from 'components/Readme';
+
 import PresentationBlock from 'components/PresentationBlock';
 
-import { 
-  DefaultExample, 
-  RoundedCornersExample, 
-  FullWidthExample, 
-  WithAnimationExample, 
-  WithLazyLoadingExample, 
-  WithLazyLoadingAndAnimationExample, 
-  BackgroundImageExample 
```

Comment:
We should really introduce some basic prettier rules for this. It already supports MDX so shouldn't be too hard.

Reply:
Good idea, I'll make a card for it.